python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -25,6 +24,13 @@
 except ImportError:
     module_logger.debug('Could not import torch, required if using a PyTorch model')
 
+try:
+    from mlflow.pyfunc import PyFuncModel
+except ImportError:
+    from typing import Any
+    PyFuncModel = Any
+    module_logger.debug('Could not import mlflow, required if using an mlflow model')
```

Comment:
Is there a unit test for this change?

Reply:
it's a dependency issue so no, but it is blocking responsibleai repository build gates because of the mlflow import error

Reply:
there is a unit test which validates the new mlflow wrapper, but not this new import logic I added to fix the other repository's gates

Reply:
actually, nvm, I just realized this can be tested by removing the mlflow dependency from requirements-dev.txt.  The existing tests would fail then if this is not working, so that would test this change.